tests/ttnn/unit_tests/module.mk

```diff
@@ -0,0 +1,29 @@
+
+
+TTNN_UNIT_TEST_DIRS := $(TT_METAL_HOME)/tests/ttnn/unit_tests/ $(TT_METAL_HOME)/tests/ttnn/unit_tests/operations/
+
+TTNN_UNIT_TESTS_SRCS := $(foreach dir,$(TTNN_UNIT_TEST_DIRS),$(wildcard $(dir)*.cpp))
+
+TTNN_UNIT_TESTS_INCLUDES := $(TT_METAL_INCLUDES) $(addprefix -I,$(TTNN_UNIT_TEST_DIRS))
```

Comment:
Personally, I would prefer we don't load up on the include directives here, but if it makes your life much easier when writing `#include`s, more power to you.